docs/usage.rst

```diff
@@ -1194,9 +1219,9 @@ Matplotlib is Unyt aware. After enabling support in :mod:`unyt` using the
 manager, Matplotlib will label the x and y axes with the units.
 
   >>> import matplotlib.pyplot as plt
-  >>> from unyt import s, K, matplotlib_support
-  >>> x = [0.0, 0.01, 0.02]*s
-  >>> y = [298.15, 308.15, 318.15]*K
+  >>> from unyt import matplotlib_support, unyt_array
+  >>> x = unyt_array([0.0, 60.0, 120.0], "s")
+  >>> y = unyt_array([298.15, 308.15, 318.15], "K")
```

Comment:
Why change how these are initialized? I think it’s more idiomatic to create a `unyt_array` via multiplication and I’d prefer it if we emphasized that in the docs.